x-pack/legacy/plugins/maps/public/meta.js

```diff
@@ -5,76 +5,85 @@
  */
 
 
-import { GIS_API_PATH, EMS_META_PATH } from '../common/constants';
-import _ from 'lodash';
-import { getEMSResources } from '../common/ems_util';
+import {
+  GIS_API_PATH,
+  EMS_CATALOGUE_PATH,
+  EMS_TILES_CATALOGUE_PATH,
+  EMS_FILES_CATALOGUE_PATH,
+  EMS_FILES_DEFAULT_JSON_PATH,
+  EMS_TILES_RASTER_TILE_PATH
+} from '../common/constants';
 import chrome from 'ui/chrome';
 import { i18n } from '@kbn/i18n';
 import { EMSClient } from 'ui/vis/map/ems_client';
 import { xpackInfo } from './kibana_services';
+import fetch from 'node-fetch';
```

Comment:
I don't see the benefit of including `node-fetch` here if this script only ever runs in the browser. `window.fetch` should already be available in 99% of browsers. Maybe not IE11 though, in which case maybe [whatwg-fetch](https://www.npmjs.com/package/whatwg-fetch) is a better polyfill.

Reply:
Will do that in separate PR. Will include for that the refactor to new platform (where we need to use the new http service)